### PR TITLE
Added documentation for benchmarking machine.

### PR DESCRIPTION
**Pull request by syan095**
Documentation update for #2042

### CODE REVIEW

File: networks/home/integration/full-node.md [hunk 1]
```diff
@@ -2,7 +2,19 @@
 
 ## Spec Requirement
 
-Same as the Kusama full node requirements.
+You can check if your machine satisfy the spec requirement by using the following make command to benchmark your machine.
+
+
+```
+make benchmark-machine
```
Comment (xlc) [inline networks/home/integration/full-node.md:9]: can we run this using docker?

**Review by xlc — APPROVED** [review]

File: networks/home/integration/full-node.md (hunk 1 above)
Comment (syan095) [reply networks/home/integration/full-node.md:14]: Added docker command. I have trouble running it on my machine, probably due to M1 chip.
Can you double check this? thanks

File: networks/home/integration/full-node.md [hunk 2]
```diff
@@ -2,7 +2,25 @@
...
 ## Spec Requirement
 
-Same as the Kusama full node requirements.
+You can check if your machine satisfy the spec requirement by using the following make command to benchmark your machine.
+
+
+
+```
+// If you are running docker image:
+docker run acala/karura-node:latest benchmark machine
```
Comment (xlc) [inline networks/home/integration/full-node.md:11]:
```suggestion
docker run acala/karura-node:latest benchmark machine --chain=karura
```

**Review by Lawrence-Acala — COMMENTED** [review]
Looks good.

**Pull request merged**